amqp-common/src/main/java/uk/nhs/adaptors/amqp/SubscribeQueue.java

```diff
@@ -0,0 +1,29 @@
+package uk.nhs.adaptors.amqp;
+
+import javax.jms.Message;
+
+import org.springframework.jms.annotation.JmsListener;
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import lombok.AllArgsConstructor;
+import lombok.SneakyThrows;
+
+@Component
+@AllArgsConstructor
+public class SubscribeQueue {
```

Comment:
subscribes should probably be part of each app that handles messages
amqp-common should only common generic code for handling queues related operations